src/databricks/labs/remorph/config.py

```diff
@@ -14,7 +14,8 @@ class TranspileConfig:
     __file__ = "config.yml"
     __version__ = 1
```

Comment:
@ericvergnaud  whenever we add new fields we bump up the version number. and build an upgrade script that will upgrade to a newer version. which can be separate PR.

Reply:
Agreed in general, but this doesn't apply here because "config.yml" is just a test sample.
What we are versioning is the data structure sent to the remote server, and they don't have access to our code base, so we need a version number in the structure itself.

Reply:
ok, but that mechanism seems to apply to server-side stuff ? How do I upgrade a local `TranspileConfig` _before_ `Installation` tries (and probably fails) to load it ?

Reply:
unbumped the version because it breaks the test
are we ok to do this in a separate PR ?
Created issue https://github.com/databrickslabs/remorph/issues/1374 to remember doing it...

Reply:
Bumped the version and updated the expected version in the failing tests